src/types/NFTInterface.ts

```diff
@@ -207,6 +210,11 @@ type MarketFixedPriceEvent = SharedMarketEventData & {
   side: FIXED_SIDE_TYPES;
   buyer?: ETHAddress;
   seller?: ETHAddress;
+  // TODO: make price data non-optional
+  usd?: CurrencyAmount;
```

Comment:
Perhaps:
```
price: {
  price: {}
  usdcPrice: {}
  nativePrice: {}
}
```

as opposed to this to make what the different types are listed correspond to

Reply:
For optionality, perhaps price? root object is optional. price within is required and usdc/native prices are optional.

Reply:
```
price?.amount
price?.symbol
price?.usdcPrice?.symbol
price?.nativePrice?.symbol
```

eg: `5 FWB`

<PriceDisplayComponent price={event.price} />

// !! TYPEERROR
<PriceDisplayCompoennt price={event.amount} currency={event.currency} />